src/sdk/python/rtdip_sdk/pipelines/transformers/spark/machine_learning/cols_to_vector.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2024 RTDIP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.ml.feature import VectorAssembler
+from pyspark.sql import DataFrame
+from ...._pipeline_utils.models import Libraries, SystemType
+from ...interfaces import TransformerInterface
+
+
+class ColsToVector(TransformerInterface):
```

Comment:
ColumnsToVector

Reply:
This is handled here: https://github.com/amosproj/amos2024ws01-rtdip-data-quality-checker/pull/81